src/main/java/de/retest/recheck/ui/actions/ActionStateSequence.java

```diff
@@ -45,10 +45,11 @@ private ActionStateSequence() {
 
 	public ActionStateSequence( final SutState initialState, final List<ActionState> actionStates, final String name,
 			final long sumDuration, final long startupDuration ) {
+		if ( initialState == null || actionStates == null ) {
+			throw new IllegalArgumentException( "initialState and actionStates cannot be null." );
```

Comment:
Please throw a correct NPE. Or use lombok `@NonNull` with a lombok constructor. 

This class should not be used anymore!?